backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -275,7 +275,10 @@ object ColumnarPartialProjectExec {
     if (expr == null) return false
     expr match {
       case _: ScalaUDF => true
-      case h if HiveUdfUtil.isHiveUdf(h) => true
+      case h
+          if HiveUdfUtil.isHiveUdf(h) &&
```

Comment:
Just a comment on this existing code. The `isHiveUdf` covers  UDAF and UDTF, which seems meaningless here for partial project. Maybe, just use `HiveUDFTransformer.isHiveUDF`?